src/PackageArchive/Scenario.ConsoleApp/Scenario.ConsoleApp.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <MaxImplicitVersion>2.1.0</MaxImplicitVersion>
+    <!-- Use pre-release versions up until 2.1.0, then don't lift higher than "2.1.0" -->
+    <MicrosoftNETCoreAppPackageVersion Condition="$(MicrosoftNETCoreApp21PackageVersion.StartsWith('$(MaxImplicitVersion)-'))">$(MicrosoftNETCoreApp21PackageVersion)</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>$([MSbuild]::ValueOrDefault('$(MicrosoftNETCoreAppPackageVersion)','$(MaxImplicitVersion)'))</MicrosoftNETCoreAppPackageVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.App" Version="$(MicrosoftNETCoreAppPackageVersion)" />
```

Comment:
Is it possible to combine this with the webapp?

Reply:
Maybe, but i wanted this refactoring to be explicitly clear about the types of projects and the scenarios that the offline cache supports. I think the code has been unclear about this in the past because we were trying too hard to optimize.

Reply:
I guess the reason why I'd want to combine this with WebApp is that the web app will also contain an implicit reference to NETCore.App as well. Class library is completely different so I can see why that's a separate scenario.